### ISSUE TITLE
Unmarshal failed for Rabbitmq 3.5.6

### ISSUE THREAD

**Issue by snowsky** [michaelklishin/rabbit-hole#97]
From https://github.com/hashicorp/terraform/issues/12810:
```
rabbitmq_exchange.test_exchange: Refreshing state... (ID: Test@BIND...ELOPMENT)
Error refreshing state: 1 error(s) occurred:

* rabbitmq_queue.test_queue: rabbitmq_queue.test_queue: json: cannot unmarshal string into Go struct field DetailedQueueInfo.consumer_utilisation of type float64
```

**Comment by michaelklishin** [comment]
Sorry but that's too little information to work with. A traffic capture is absolutely necessary when investigating this sort of issues.

HTTP API format does change over time, sometimes is fairly subtle ways that only a few clients .are affected by. Supporting RabbitMQ 3.5.x is a low priority for `rabbit-hole`.

**Comment by michaelklishin** [comment]
As mentioned in the Terraform issue, this is https://github.com/rabbitmq/rabbitmq-management/issues/26, which was fixed in RabbitMQ 3.5.7.
